application/common/models/Mfa.php

```diff
@@ -566,4 +574,9 @@ public function setLabel($label)
             $this->label = $this->getReadableType();
         }
     }
+
+    public static function removeOldManagerMfaRecords(): void
+    {
+        self::deleteOldRecords('1 week', ['type' => Mfa::TYPE_MANAGER]);
```

Comment:
I'm not sure I understand... this looks like it will not let anyone use the "manager" MFA option if their IdP account is more than a week old. Is that intentional? I could very well be misreading this, so please feel free to correct my understand of the code.

Reply:
The date it's using is the `mfa` `created_utc` date, not the `user` created date (which, strangely, does not exist).

Reply:
Okay. When is the "manager" MFA option added? I thought that was a built-in option added automatically as long as the user didn't have any other MFA options.

Reply:
It is added when the user selects "I need help" on the 2sv "More options" popup.

Reply:
Ohhh! Thanks. This makes so much more sense now.